.tekton/tasks/test-groups/test-ci-autoprofile-task.yaml

```diff
@@ -72,6 +76,12 @@ spec:
         CIRCLE_COMPARE_URL=""
         MODIFIED_FILES=""
         GIT_COMMIT_DESC=""
+        if [ "$(params.prerelease)" == "true" ]; then
+          bin/install-prerelease.sh "$(params.prerelease-node-version)"
+          source "$HOME/.nvm/nvm.sh"
+          nvm use "$(params.prerelease-node-version)" > /dev/null 
```

Comment:
qs: Why are these comments not in the script? 🤔 

> source "$HOME/.nvm/nvm.sh"
> nvm use "$(params.prerelease-node-version)" > /dev/null

Reply:
We are executing `bin/install-prerelease.sh "$(params.prerelease-node-version)" from test suits, which installs the required node version.
Then we are just using the required version from the individual files to make sure that there is no conflict with the image version and installed version.

Reply:
But we already call
> source "$HOME/.nvm/nvm.sh"

In the script? 🤔 Why do we have to do that again?